api/resources/quests.py

```diff
@@ -8,11 +8,11 @@
 from sqlalchemy.orm.exc import NoResultFound
 
 from api import db
-from api.database.models import Quest
-from api.database.models import User
-from api.database.models import UserQuest
-from api.database.models import Encounter
-from api.database.models import Action
+from api.database.models.quests import Quest
+from api.database.models.users import User
+from api.database.models.user_quests import UserQuest
+from api.database.models.encounters import Encounter
+from api.database.models.actions import Action
```

Comment:
We should see if we can add this to  a `resources helper` or something like that

Reply:
Great idea!